### DISCUSSION TITLE
Raspberry Pi4

### DISCUSSION THREAD

**Discussion by actraiser30** [86Box/86Box#3193] [Q&A] [answered]
Hello

how can i build this for the Raspberry Pi 4?

i cannot find any info about

thanks

**Comment by parkerlreed** [comment]
Are you needing to compile it for a specific reason? There's already ARM64 release builds that work for the Raspberry Pi

https://github.com/86Box/86Box/releases/download/v3.11/86Box-NDR-Linux-arm64-b4311.AppImage

**Comment by symbios24** [comment]
Thanks but do i install this appimage file to linux???

**Comment by parkerlreed** [comment, marked as answer]
You download it to the Pi and run it as any other executable

```
chmod +x 86Box-NDR-Linux-arm64-b4311.AppImage
./86Box-NDR-Linux-arm64-b4311.AppImage
```

**Reply by symbios24** [reply]
The appimage file does not work/run it says Apprun error not such a file or directory

**Reply by parkerlreed** [reply]
> The appimage file does not work/run it says Apprun error not such a file or directory

That means you are likely on 32bit Raspberry Pi OS and this won't work there. 

The binary is ARM64 only and can't load on 32bit OS.

**Reply by symbios24** [reply]
I tried both binaries 32 and 64 bits got the same error Its not working

**Comment by actraiser30** [comment]
any change to make this work in Raspberry pi 4??
